.evergreen/.evg.yml

```diff
@@ -1920,8 +1920,8 @@ buildvariants:
     - name: "test"
 
 - matrix_name: "tests-jdk-secure"
-  matrix_spec: { auth: "auth", ssl: "ssl", jdk: ["jdk8", "jdk17"], version: ["3.6", "4.0", "4.2", "4.4", "5.0", "6.0", "7.0", "latest"],
-                 topology: ["replicaset", "sharded-cluster"], os: "linux" }
+  matrix_spec: { auth: "auth", ssl: "ssl", jdk: [ "jdk8", "jdk17" ], version: [ "3.6", "4.0", "4.2", "4.4", "5.0", "6.0", "7.0", "latest" ],
+                 topology: "*", os: "linux" }
   display_name: "${version} ${topology} ${auth} ${ssl} ${jdk} ${os} "
   tags: ["tests-variant"]
   tasks:
```

Comment:
Not directly related to the changes made in this pull request,  however, I noticed that the `tests-jdk-secure-jdk11` matrix could potentially be consolidated with the `tests-jdk-secure` matrix. The latter runs tests using JDK11 against version 7.0. It might be worth considering running JDK11 against all versions and topologies. However, I understand there may be specific reasons why JDK11 is currently limited to version 7.0 and replicaset topology, which I may not be aware of